libflux/flux-core/src/formatter/tests.rs

```diff
@@ -1268,3 +1268,17 @@ a",
     );
     assert_unchanged(r#"a = "a string literal with a tabstop '	'""#);
 }
+
+#[test]
+fn invalid_syntax() {
+    assert_unchanged(
```

Comment:
The `assert_unchanged` function will panic on errors. I think you should just call `format` here and check that you get an error.